public/docs/_examples/server-communication/dart/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   http: ^0.11.3+3
   jsonpadding: ^0.1.0
   stream_transformers: ^0.3.0+3
-  http_in_memory_web_api: ^0.0.1
+  http_in_memory_web_api: ^0.2.0
```

Comment:
@wardbell : there was a bug in `http_in_memory_web_api` which the author fixed. This PR removes the temporary workaround and bumps up the library version to get the fixed version.